arkouda_workflows/README.md

```diff
@@ -107,62 +134,45 @@ The label matching one of the Prometheus matchLabels elements is specified in th
 The [deploy-arkouda-on-kubernetes-command.sh](deploy-arkouda-on-kubernetes-command.sh) script is used to deploy AoK utilizing several environment variables. In the base configuration Arkouda runs as the default AOK user,  an example of which is shown below:
 
 ```
-export KUBERNETES_USER=arkouda
 export KUBERNETES_URL=https://localhost:6443
 export ARKOUDA_MEMORY=2048Mi
 export ARKOUDA_CPU_CORES=2000m
-export ARKOUDA_PROMETHEUS_MATCH_LABEL=release: kube-stack
+export CHPL_NUM_THREADS_PER_LOCALE=2
+export CHPL_MEM_MAX=1000000000
+export ARKOUDA_SERVICEACCOUNT_NAME=arkouda-on-k8s 
+export ARKOUDA_SERVICEACCOUNT_TOKEN_NAME=arkouda-on-k8s 
 export ARKOUDA_SSH_SECRET=arkouda-ssh
-export ARKOUDA_SSL_SECRET=arkouda-tls
 export ARKOUDA_NAMESPACE=arkouda
+export ARKOUDA_NUMBER_OF_LOCALES=2
+export ARKOUDA_TOTAL_NUMBER_OF_LOCALES=3
 export ARKOUDA_METRICS_SERVICE_PORT=5556
-export ARKOUDA_VERSION=v2024.02.02
+export ARKOUDA_VERSION=v2024.04.19
```

Comment:
this version is `v2024.04.19` but every other version in this PR is `v2024.03.18`